dvm-helper/dockerversion/dockerversion.go

```diff
@@ -102,26 +102,26 @@ func (version Version) buildDownloadURL(mirror string, forcePrerelease bool) (ur
 // version - the desired version.
 // mirrorURL - optional alternate download location.
 // binaryPath - full path to where the Docker client binary should be saved.
-func (version Version) Download(mirrorURL string, binaryPath string, l *log.Logger) error {
-	err := version.download(false, mirrorURL, binaryPath, l)
+func (version Version) Download(opts config.DvmOptions, binaryPath string) error {
+	err := version.download(false, opts, binaryPath)
```

Comment:
Something @captainsafia pointed out to me is that we should [avoid the boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap). Since this was there before, I wouldn't bother changing it now. Might be good though, as I don't know if this means it's downloading or not.